server/routes/utils/reportHelper.ts

```diff
@@ -70,22 +76,25 @@ export const generatePDF = async (
       headless: true,
     });
     const page = await browser.newPage();
-    await page.goto(url, { waitUntil: 'networkidle0' });
 
+    await page.setDefaultNavigationTimeout(0);
     await page.setViewport({
       width: windowWidth,
       height: windowHeight,
     });
 
+    await page.goto(url, { waitUntil: 'networkidle0' });
+
     const timeCreated = new Date().toISOString();
-    const fileName = getFileName(itemName, timeCreated);
+    const fileName = getFileName(itemName, timeCreated) + formats.PDF;
     // The scrollHeight value is equal to the minimum height the element would require in order to fit
     // all the content in the viewport without using a vertical scrollbar
     const scrollHeight = await page.evaluate(
       () => document.documentElement.scrollHeight
     );
 
     const buffer = await page.pdf({
+      path: fileName,
```

Comment:
We are not saving any actual file, remove this option.

Reply:
Will remove